File: 1036 - Bhaskara's Formula/bhaskara.cpp
#include <bits/stdc++.h>
using namespace std;
int main(){
    double a, b, c;
    cin >> a >> b >> c;
    double delta = b*b - 4*a*c;
    if(delta < 0 || a == 0) cout << "Impossivel calcular" << endl;
    else {
        printf("R1 = %.5f\n", (-b + sqrt(delta)) / (2*a));
        printf("R2 = %.5f\n", (-b - sqrt(delta)) / (2*a));
    }
}

File: 1036 - Bhaskara's Formula/bhaskara.py
import math
a, b, c = map(float, input().split())
delta = b*b - 4*a*c
if delta < 0 or a == 0:
    print("Impossivel calcular")
else:
    print("R1 = {:.5f}".format((-b + math.sqrt(delta)) / (2*a)))
    print("R2 = {:.5f}".format((-b - math.sqrt(delta)) / (2*a)))
